plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -192,7 +221,14 @@ public DistributedQueryRunner build()
 
             try {
                 queryRunner.installPlugin(new TpchPlugin());
-                queryRunner.createCatalog("tpch", "tpch");
+                Map<String, String> tpchCatalogProperties = ImmutableMap.<String, String>builder()
+                        .put("tpch.column-naming", tpchColumnNaming.name())
+                        .put("tpch.double-type-mapping", tpchDecimalTypeMapping.name())
+                        .buildOrThrow();
+                queryRunner.createCatalog("tpch", "tpch", tpchCatalogProperties);
+
+                queryRunner.installPlugin(new TpcdsPlugin());
+                queryRunner.createCatalog("tpcds", "tpcds");
```

Comment:
why do we think it's generally useful?

Reply:
`HiveQueryRunner#main` is a convenient way to start pseudo distributed cluster for testing. Having `tpcds` connector deploy allows to run `tpcds` queries and play with the `tpcds` dataset. Is the concern here that `tpcds` will be using unnecessary resources when running integration tests? If that's the case and if we think that the resources used are significant I can make this conditional and only enable it in `HiveQueryRunner#main`

Reply:
I am more concerned about startup time. but probably it doesn't cost much.

(i almost never use tpcds data set, so i am surprised this is useful outside of benchmarks)

Reply:
We are planning to use tpcds dataset for project Tardigrade testing